### DISCUSSION TITLE
V2.77.1

### DISCUSSION THREAD

**Discussion by zhongyang219** [zhongyang219/MusicPlayer2#770] [Announcements]
此次更新修正了2.77版本的若干问题，没有增加新的功能。
* 修正在右键菜单中“添加到播放列表”子菜单有时会异常禁用的问题。
* 修正点击“AB重复”按钮时的测试文本没有删除的问题。
* 修正“AB重复”按钮鼠标提示不正确的问题。
* 修正100%缩放比例下对话框中的控件在垂直方向太挤的问题。
* 修正从全屏模式进入迷你模式时歌词文本太大并且显示不全的问题。
* 修正aac文件在属性对话框中无法显示内嵌专辑封面的问题。

<hr /><em>This discussion was created from the release <a href='https://github.com/zhongyang219/MusicPlayer2/releases/tag/V2.77.1'>V2.77.1</a>.</em>

**Comment by ttttr885** [comment]
提两个优化的点吧 第一是桌面歌词太卡了 明显的掉帧  第二是播放列表 有点儿土

**Reply by franklett** [reply]
作者说过，掉帧是因为无法使用GPU绘制，可以被改善通过减小界面刷新时间（比如从50ms到10ms）。

**Reply by iliwyke** [reply]
播放歌词与音频对不上

**Comment by franklett** [comment]
由于实际需要（在100寸4K电视上，极简地播放歌词，同时用着桌面和窗口两种形式），尝试过很多歌词软件（包括各音乐大厂的客户端，它们都不理想），最终发现这个软件（感恩作者和贡献者的付出）。

对于目前无法支持GPU绘制，尽管作者说原因在于GDI库（是不是出于平台兼容的考虑，或开源协议的限制），并且可以被改善（通过减小界面刷新时间在配置中），但还是特别希望它有升级在程序层面（比如增加一个选项：用DirectX绘制，或者其它开源引擎）。

**Reply by franklett** [reply]
再次感恩。愿这个软件继续兴旺下去。由于没学好C++和Windows编程，我只能付出这些苍白的言语。

**Comment by 1184747075** [comment]
请问目前没有平衡音量的功能吗

**Comment by ax11071039** [comment]
![image](https://github.com/user-attachments/assets/037f7f5d-6d4b-4632-9dba-16052131bb08)
调节音量怎么不是滑动条的？这种加减方式不符合使用习惯。

**Reply by SqianRay** [reply]
鼠标悬停在上面就能滑动

**Comment by SqianRay** [comment]
超牛超好用 love~ UI自定义太好了
![屏幕截图 2024-10-20 202505](https://github.com/user-attachments/assets/578f353c-8723-4ddd-bea2-6dced04be979)
一些我觉得可以增加的UI自定义功能：
1.字体自定义 - 包括字族、字号、字重；中西文分别设置字体 - UI字体能自定义的话也可以自用一些版权字体了，字重自调可以让标题醒目美观；
2.高分辨率图标 - 适配高分屏；
3.对齐功能 - 在现有字体对齐的基础上，让矩形、专封、图标都能选择对齐；
4.可选性全屏模式保留 Windows 任务栏 - 最大化窗口和全屏显示各有好处，提供更多选择；
5.全屏模式下关闭电脑休眠 - 让电脑成为提词器；
6.可选性小、窄、大窗口尺寸 - 让一些畸形的窗口根源上消失
![屏幕截图 2024-10-20 205044](https://github.com/user-attachments/assets/430f4214-c178-4a57-8e2a-8787d9f32c93)
![屏幕截图 2024-10-20 205054](https://github.com/user-attachments/assets/8c65566a-edbc-4dc1-8031-1d7e1298315c)
![屏幕截图 2024-10-20 205109](https://github.com/user-attachments/assets/1cb97549-20ae-49d3-90e0-8306209f6176)
...

**Reply by SqianRay** [reply]
7.切换列表时，如从一张专辑选到另一张专辑，是否从上次播放的记忆点继续播放，改为可选项。

**Reply by happrayha** [reply]
这个UI改的好看，能分享下设置吗

**Reply by Larewell** [reply]
> 这个UI改的好看，能分享下设置吗

![微信图片_20241029023453](https://github.com/user-attachments/assets/02cd421e-b158-4987-98b1-1d915f9e5125)
这个也好看，选第二个就好啦

**Comment by SqianRay** [comment]


**Reply by happrayha** [reply]
看不到，能发xml吗

**Reply by SqianRay** [reply]
[nowimuds.zip](https://github.com/user-attachments/files/17531530/nowimuds.zip)

**Comment by winmetro** [comment]
当你用这个软件区播放内嵌歌词的文件的时候，他会把歌词提出来，弄得歌曲的文件夹里乱成一团糟。此外有些歌词显示不正常，或不完整。在千千静听上就做的不错（除了没翻译)望改正。

**Comment by 371606442** [comment]
win10 22H2 19045.5198,启用搜索栏歌词的时候会显示一下然后直接就透明了，把所有任务栏透明的插件退出后仍然无法解决，实在是没办法国。

**Comment by ootdyao** [comment]
可以增加自定义图标吗？万分感谢

**Comment by q2994500546** [comment]
首先希望可以win11适配任务栏歌词。
存在问题：
1、桌面歌词拖到任务栏上锁定，点击任务栏的图标就会被任务栏覆盖掉，导致看不到歌词，切歌新歌词加载后会重新回到顶层，但是再次点击任务栏图标会再次被覆盖。
2、win11默认搜索框无法使用任务栏歌词
3、使用startallback的《增强型经典任务栏》功能后可以在搜索框位置显示歌词。但存在两个问题：
     1.光标在歌词上滑动时会黑白频闪；
![2025-08-30 18-15-07](https://github.com/user-attachments/assets/3ca88634-be88-4783-a662-fb66a8ffed5b)
     2.任务栏为透明状态时，歌词会有白边。
<img width="1574" height="156" alt="歌词白边" src="https://github.com/user-attachments/assets/d7117405-759a-4dcb-8b87-f981a0657209" />
     系统：win11企业版23H2
建议：
1、点击右上角退出希望可以是隐藏播放器至后台，而不是直接退出。《显示/隐藏播放器》这个功能只能用快捷键有点不太方便
2、任务栏歌词可以不放在搜索框，定位锁定到任务栏的左下角或者其他位置都可以，并且鼠标滑动到歌词上面可以切换为播放控件。（参考酷狗可以放在任务栏左下角的，但那个太丑了，字体都调不了）

**Comment by snkplaymore123456-lang** [comment]
您好!软件的作者, 我很喜欢这个播放器播放MIDI音乐很棒,可以自定义音色库~ 
想问一下, 哪个版本支持在XP系统上运行呀?

**Reply by zhongyang219** [reply]
> 您好!软件的作者, 我很喜欢这个播放器播放MIDI音乐很棒,可以自定义音色库~ 想问一下, 哪个版本支持在XP系统上运行呀?

最后一个支持XP系统的版本是2.66，可以从release页面下载，但是早期版本功能相对比较简陋。

**Comment by LlReww** [comment]
带佬，我这不知道为啥搜不到歌词啥的了，是wyu接口掉了吗？怎么修啊大佬求告知，这个软件真的很好用，均衡器也很舒服
<img width="581" height="599" alt="屏幕截图 2025-12-15 134908" src="https://github.com/user-attachments/assets/fe1fbfcc-05f3-42cf-bfa4-1394a349f5d1" />

**Reply by zhongyang219** [reply]
已经解决了，可以到Action页面下载最新的预构建版本，或者等待更新，近期应该会发布新版本了

**Reply by LlReww** [reply]
> 已经解决了，可以到Action页面下载最新的预构建版本，或者等待更新，近期应该会发布新版本了

谢谢你大佬，爱你😘
